hack/build/docker/builder/Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN sed -i 's/proxy = None//gI' /tmp/fedora_ci.dnf.repo && \
 	patch \
 	libnbd-devel \
 	nbdkit-devel \
+        nbdkit-curl-plugin \
```

Comment:
@awels because it is used to run the unit tests. I added some tests in the `pkg/image/nbdkit_test.go` without these deps, it cannot find the plugin/filter *.so

Reply:
This is also the reason why I need to upgrade the `kubevirt-cdi-bazel-builder` image to fedora 33 (#1511)

Reply:
So this is the reason we have an interface QEMUOperations for the qemu operations that currently exist. We can mock out the actual calls to qemu by implementing a mock struct that implements all the functions. In those functions we can then verify that we are receiving the correct values and we can return any potential errors without having to worry about if the system running the tests actually has qemu or not. It also makes the test faster as it doesn't have to fork a process to run qemu.

Reply:
Oh ok, I see!  I haven't noticed that! I'll change the code accordingly! Many thanks!

Reply:
Removed those deps